validator/client/redis/producer.go

```diff
@@ -76,19 +79,17 @@ func NewValidationClient(cfg *ValidationClientConfig) (*ValidationClient, error)
 		return nil, err
 	}
 	return &ValidationClient{
-		name:           cfg.Name,
```

Comment:
I might've missed something, but I don't think `name` or `producerConfig` are populated anymore after this diff. I think they were meant to be replaced with `c.config.Name/ProducerConfig` but they don't seem to have been replaced with that.